src/main/resources/db/migration/h2/V2_0__add_user_tables.sql

```diff
@@ -1,12 +1,12 @@
-CREATE TABLE audit_user
+CREATE TABLE staff.audit_user
```

Comment:
If this migration is common between h2 and postgres can we stick it in a `migration/common` and avoid duplication?